stories/assets/js/parallax_lines.js

```diff
@@ -0,0 +1,25 @@
+import './viewport';
```

Comment:
@pallavikadam-srijan Can we change the file name to just be `parallax.js`? Also can we use some class other than `.ourexpertise`, since if the class is changed/removed in future then the Js won't init.